app/actions/views/login.js

```diff
@@ -39,9 +39,6 @@ export function handleSuccessfulLogin() {
             }
         }, getState);
 
-        Client.setToken(token);
```

Comment:
Are we not using the regular Client at all for anything now, or is this set somewhere else?

Reply:
We don't use the regular Client at all in the RN app